src/index.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-unused-vars */
+import _ from 'lodash';
+import './style.css';
+import Data from './data.xml';
+import Notes from './data.csv';
+
+const section = document.querySelector('section');
+const taskInput = document.querySelector('input');
+const tasksContainer = document.querySelector('.tasks-container');
+const completedTasks = document.querySelector('button');
+
+class myList {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+const listArray = [];
+const addTask = (taskValue) => {
+  const newTaskContainer = document.createElement('div');
+  newTaskContainer.classList.add('new-task--container');
+  newTaskContainer.innerHTML += `
+   <div class="left-content">
+      <input type="checkbox" class="check-task">
+      <span>${taskValue}</span>
+   </div>
+   <div class="right-content">
+      <i class="fa-solid fa-ellipsis-vertical"></i>
+      <i class="fa-solid fa-trash-can"></i>
+   </div>`;
+  tasksContainer.appendChild(newTaskContainer);
+};
+taskInput.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter' && taskInput.value) {
+    e.preventDefault();
+    addTask(taskInput.value);
+    taskInput.value = null;
+  }
+});
```

Comment:
**[REQUIRED CHANGES]**

- Hey, you've done great implementing the to-do list structure however, I think your implementation is great but still haven't followed the project requirement. Project requirements are conditions that must be met by a solution component to maintain standards and conforms to the laid down styles or pattern. 
When comparing your implementation with the [project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md), It's obvious that you've implemented it but your solution is  not conforming to all the requirement especially this part of the image below:

:point_down: 
*missing requirement*
![requirements](https://user-images.githubusercontent.com/59397159/173101684-c9876778-19a1-4247-b81f-75bcda0b289e.png)

- As you can see, your todos are meant to be filled and populated dynamically on page load, but right now, it's empty until a user adds some list of todos which is technically not meeting the project requirement for this milestone.
Please, let's try to follow the project requirement irrespective of how experienced we are as this would help in keeping things simple :pray: :pray: